src/css/style.css

```diff
@@ -511,4 +512,104 @@ nav {
   }
 }
 
-/* footer section */
+/* footer section styles */
+footer {
+  position: relative;
+}
+
+.re-position {
+  margin-block: 30px;
+  position: absolute;
+  width: 100%;
+  top: -120px;
+  background: url("../images/footer-shap.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  z-index: 1;
+}
+
+.footer-form {
+  display: flex;
+  flex-direction: column;
+  background: url("../images/footer-mobile.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: top right;
+  padding-block: 6em;
+  margin-block: 20px;
+}
+
+.text-white {
+  color: #fff;
+  text-align: center;
+}
+
+.p-size {
+  font-size: 20px;
+  max-width: 47ch;
+  margin: auto;
+  padding-inline: 1.4em;
+}
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 1em;
+  margin-top: 1.4em;
+  max-width: 600px;
+  margin-inline: auto;
+}
+
+.contact-form input,
+textarea {
+  width: 100%;
+  padding: 0.5em;
+}
+
+.contact-form ::placeholder {
+  font-size: 0.9em;
+  color: var(--color-light-gray);
+}
+
+.contact-form input:is(:focus),
+textarea {
+  outline: none;
+}
+
+.contact-button {
+  display: inline-block;
+  width: 140px;
+  margin-left: 0;
+}
+
+@media screen and (min-width: 40em) {
+  .contact-form {
+    width: 100%;
+    max-width: 430px;
+  }
+
+  .contact-button {
+    margin-inline: auto;
+  }
+
+  .re-position {
+    margin-block: 30px;
+    position: absolute;
+    width: 100%;
+    top: -120px;
+    background: url("../images/footer-shap.png");
+    background-repeat: no-repeat;
+    background-position: top right;
+    background-size: cover;
+    z-index: 1;
+  }
+
+  .footer-form {
```

Comment:
- [ ] Great job so far on your form 👏🏻 However there is one missing requirement, kindly make sure to address this:

- According to the project requirements, Button interactions such as hover, active (when it's pressed), disabled, and enable (default state) are styled as shown in the Figma template.
 - Please use this image below 👇🏻 to improve the background color effect of your  `Get in touch` button 👍🏻 

![image](https://user-images.githubusercontent.com/79058364/186333331-cc91ebb0-aaa9-4d12-a5d0-c731326018f9.png)